docs/developers-guide/README.md

```diff
@@ -31,9 +31,9 @@ Finally, it provides a step by step walk-throughs on how various Morphir compone
     1. [Elm Frontend](#) <br>
     2. [Elm Incremental Frontend](#) <br>
 10. [The Morphir Backends](#) <br>
-    1. [Scala Backend](https://github.com/finos/morphir-elm/blob/documentation/docs/developers-guide/files/scala-backend.md)
-    2. [Spark Backend](https://github.com/finos/morphir-elm/blob/documentation/docs/developers-guide/files/spark-backend.md)
-    3. [Relational IR Backends](https://github.com/finos/morphir-elm/blob/documentation/docs/developers-guide/files/relational-backend.md)
+    1. [Scala Backend](https://github.com/finos/morphir-elm/blob/main/docs/developers-guide/files/scala-backend.md)
+    2. [Spark Backend](https://github.com/finos/morphir-elm/blob/main/docs/developers-guide/files/spark-backend.md)
+    3. [Relational IR Backends](https://github.com/finos/morphir-elm/blob/main/docs/developers-guide/files/relational-backend.md)
```

Comment:
Would it make sense to make all the links relative instead?

Reply:
Sure, its much better. Has been effected now.